src/sentry/dynamic_sampling/rules/base.py

```diff
@@ -68,27 +67,16 @@ def get_guarded_blended_sample_rate(organization: Organization, project: Project
     if sample_rate == 1.0:
         return float(sample_rate)
 
-    # For now, we will keep this new boost for orgs with the sliding window enabled.
-    #
     # In case the organization or the project have been recently added, we want to boost to 100% in order to give users
-    # a better experience. Once this condition will become False, the dynamic sampling systems will kick in.
+    # a better experience. Once this condition will become false, the dynamic sampling systems will kick in.
     if is_recently_added(model=project) or is_recently_added(model=organization):
         return 1.0
 
-    # We want to use the normal sliding window only if the sliding window at the org level is disabled.
-    if is_sliding_window_enabled(organization):
```

Comment:
Sliding window is currently unused, so I will just remove it here but will create a ticket to follow up with an actual removal of the code.